release/models/ospf/openconfig-ospfv2-area-interface.yang

```diff
@@ -437,6 +492,64 @@ submodule openconfig-ospfv2-area-interface {
           }
         }
 
+        container authentication {
+          description
+            "Configuration and operational state parameters for
+            OSPFv2 authentication on the interface.";
+
+          container config {
+            description
+              "Configuration parameters for OSPFv2 authentication on
+              the interface";
+            uses ospfv2-area-interface-authentication-config;
+          }
+
+          container state {
+            config false;
+            description
+              "Operational state parameters for OSPFv2 authentication on
+              the interface";
+            uses ospfv2-area-interface-authentication-config;
+          }
+
+          container message-digests {
+            description
+              "Enclosing container for the list of message-digest keys that
+              can be configured on the interface";
+
+            list message-digest {
+              key "md-key-id";
+              max-elements 2;
```

Comment:
Why does this restriction exist?

Reply:
Looks like it was arista specific, removed it